### ISSUE TITLE
[Config] TOML Configs for the catalina-server.xml

### ISSUE THREAD

**Issue by YvonneW** [wso2/product-is#6602]
Need the corresponding TOML configs for the following in the `catalina-server.xml` file as it is referred in [1].

```
<Valve className="org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve" threshold="600"/>
```

[1] https://is.docs.wso2.com/en/5.9.0/administer/performance-tuning/#setting-the-thread-execution-limit
[2] https://is.docs.wso2.com/en/5.9.0/administer/performance-tuning-recommendations/

**Comment by senthalan** [comment]
This valve is now available by default in the 5.9.0 

https://github.com/wso2/product-is/blob/master/modules/distribution/src/repository/resources/conf/templates/repository/conf/tomcat/catalina-server.xml.j2#L73


But the threshold is not templated as of now

**Comment by isharak** [comment]
This issue is being closed due to extended inactivity. Please feel free to reopen it if further attention is needed. Thank you for helping us keep the issue list relevant and focused!
